Gemfile

```diff
@@ -13,11 +13,9 @@ gemspec
 # Koi
 gem 'koi_config'                , git: 'https://github.com/katalyst/koi_config.git'
 
-# i18n ActiveRecord backend
-gem 'i18n-active_record'        , git: 'https://github.com/svenfuchs/i18n-active_record.git',
-                                  branch: 'master'
-
-gem 'sass-rails', '~> 5.0.0'
-gem 'ornament', github: 'katalyst/ornament'
+gem 'ornament', github: 'katalyst/ornament', branch: 'feature/koi-3-rails-upgrade'
```

Comment:
We will need to record somewhere to change this to master when ornament is released properly.